### ISSUE TITLE
Enable/Disable LED (LON) is always set to ON state

### ISSUE THREAD

**Issue by Martin-100** [dragino/TrackerD#11]
Common.cpp -> void SYS::config_Read(void) ->contains on line 618-622:

//lon
  lon = DATA.readUChar(addr1);                   
  addr1 += sizeof(unsigned char);  
  if(lon == 0)
    lon = 1;

So writing a 0 to disable the leds results in turning them back on again?

**Comment by mkwillis** [comment]
Can this issue be fixed by editing the code?.... Its a highly annoying LED

**Comment by namejinnameyuan** [comment]
Yes, you can compile the code to change it. I changed it in v1.4.4. Before, 1 was on and 2 was off.

**Comment by P-etr** [comment]
Hi, my tracker has FW 1.4.4. and AT+LON keeps returning to 1.
So I can't turn off the LED. Is it possible to somehow turn off the LED now? Will this be fixed in future versions?

**Comment by mkwillis** [comment]
Yes LED settings DO NOT stick... We need a fix pls

**Comment by namejinnameyuan** [comment]
Hi
I will release a new version in the next few days, fix problems, and add features will be described




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "dragino/TrackerD"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年4月23日(星期天) 晚上8:04
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [dragino/TrackerD] Enable/Disable LED (LON) is always set to ON state (Issue #11)





 
Hi, my tracker has FW 1.4.4. and AT+LON keeps returning to 1.
 So I can't turn off the LED. Is it possible to somehow turn off the LED now? Will this be fixed in future versions?

**Comment by namejinnameyuan** [comment]
Hi
I will release a new version in the next few days, fix problems, and add features will be described



------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "dragino/TrackerD"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年4月23日(星期天) 晚上9:18
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [dragino/TrackerD] Enable/Disable LED (LON) is always set to ON state (Issue #11)





 
Yes LED settings DO NOT stick... We need a fix pls
